schedule/yam/agama.yaml

```diff
@@ -4,8 +4,10 @@ description: >
   Perform interactive installation with agama.
 schedule:
   - yam/agama/boot_agama
+  - yam/agama/import_agama_profile
   - yam/agama/patch_agama_tests
   - yam/agama/agama
+  - yam/agama/agama_reboot
   - installation/grub_test
```

Comment:
For s390x, the grub_test is not needed.

Reply:
if so, we'd add `- installation/grub_test` to `agama_reboot.pm` like:
```
    if (is_s390x) {
        reconnect_mgmt_console;
    } else {
        autotest::loadtest 'installation/grub_test';
    }
```
do that answer your concern? @lemon-suse